Modules/RaidManager/RaidManager.lua

```diff
@@ -305,6 +314,40 @@ function RaidManager:EndRaid(raid)
     --     LOG:Message("You are not in an active raid.")
     --     return
     -- end
+
+    -- Rebuild player list for current status of raid
```

Comment:
this should not be needed here at all. the raid status comes from ledger entries. probably user info sourcing comes from wrong place